pkg/provisioner/fixture/fixture.go

```diff
@@ -16,6 +16,15 @@ var deprovisionRequeueDelay = time.Second * 10
 var provisionRequeueDelay = time.Second * 10
 var inspectionRequeueDelay = time.Second * 2
 
+const (
+	// DefaultRequeueSecs is the default number of seconds to wait before requeuing
+	DefaultRequeueSecs    = 5
+	DefaultSizeBytes      = 93
+	DefaultClockMegahertz = 3.0
+	DefaultRAMMebibytes   = 128
+	DefaultGB             = 1024
```

Comment:
Please use lowercase initial letters unless actually using these from other packages. The capital initial letter means that the constant is exported for use outside of this package, which we do not want to do in general unless there is some reason for it.

Reply:
After utilizing these for the `demo`, it makes sense to be able to export these now.